lab-raziyeh/app/component/gallery/create-gallery/create-gallery.html

```diff
@@ -0,0 +1,20 @@
+
+    <form name="createGalleryForm" class="form-inline" novalidate>
+      <div class="form-group">
+        <input name="name" class="form-control" type="text" uib-tooltip="name required"
+          ng-model="createGalleryCtrl.gallery.name"
+          placeholder="name"
+          required>
+      </div>
+
+      <div class="form-group">
+        <input name="desc" class="form-control" type="text" uib-tooltip="description required"
+          ng-model="createGalleryCtrl.gallery.desc"
+          placeholder="description"
+          required>
+      </div>
+      
+      <button class="btn btn-default" type="submit" ng-click="createGalleryCtrl.createGallery()"> 
+        <span class="glyphicon glyphicon-plus"></span> Add Gallery 
+      </button>
+    </form>
```

Comment:
@Raziyehbazargan your markup in this file is heavily indented - maybe an editor issue?